.codecov.yml

```diff
@@ -0,0 +1,7 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 50%
```

Comment:
Is the plan to gradually update this value to a higher one as we add more coverage?

Reply:
Yes that's definitely the plan ! I will leave it to you :)